content/product/product_management/index.md

```diff
@@ -24,9 +24,11 @@ Launch tier levels are also an important concept to be aware of, and the definit
 
 ## Communications
 
-Product team members are empowered to communicate directly with customers, internal stakeholders, or anyone else. Sometimes it can be helpful to have someone review your communication before sending it, especially if it is going to a wide or unfamiliar audience. If you want, the Marketing and PR teams are available to help any time: simply ask in #marketing and someone will be happy to review your communication.
+Communications around feature roll-out are especially important, and are documented on[the rollout process](./rollout_process.md#communications) page.
 
-## Release early, release often
+In general, product team members are empowered to communicate directly with customers, internal stakeholders, or anyone else. Sometimes it can be helpful to have someone review your communication before sending it, especially if it is going to a wide or unfamiliar audience. If you want, the Marketing and PR teams are available to help any time: simply ask in #marketing and someone will be happy to review your communication.
```

Comment:
Most of these make sense, just a couple clarifications:

- On posting YouTube videos: we record demos and things like that. Why do these need special review? What's the worry that will happen? It feels to me pretty low risk.
- Can you be more specific on what anything related to press would mean, and social? I assume this doesn't mean no employees can mention Sourcegraph on social media, does it just mean tweeting from the official accounts?
- Lastly, you mentioned it doesn't have to be marketing but marketing should review - assuming it's a typo but just wanted to confirm what the typo was? I'm assuming the second marketing meant "someone". Were you thinking someone like the product directors? Or did you have a set of reviewers in mind?

Reply:
Here's a proposal, pending feedback on the above.

```suggestion
In general, product team members are empowered to communicate directly with customers, internal stakeholders, or anyone else. Sometimes it can be helpful to have someone review your communication before sending it, especially if it is going to a wide or unfamiliar audience. If you want, the Marketing and PR teams are available to help any time: simply ask in #marketing and someone will be happy to review your communication.

There are just a few places where a review is required; these can be done by someone from the marketing team, or by a product director:

- Release posts/blog posts (this is already part of the editorial process)
- YouTube videos
- About site updates
- Comms related to pricing/packaging changes
- Comms related to feature deprecation
- Public presentations/events/speaking engagements
- Anything related to press
- Social (everything should be going through Kristen, so this should be covered)
```

Reply:
Thanks @abschu! I've applied the recommendations, please take a look and let me know if anything looks wrong. In the meantime I'll go ahead and merge this.